group_vars/all/logrotate.yml

```diff
@@ -17,4 +17,4 @@ logrotate_scripts:
         if [ -d /etc/logrotate.d/httpd-prerotate ]; then \
               run-parts /etc/logrotate.d/httpd-prerotate; \
             fi \
-      postrotate: invoke-rc.d nginx rotate >/dev/null 2>&1
+      postrotate: "[ -f /var/run/nginx.pid ] && kill -USR1 `cat /var/run/nginx.pid`"
```

Comment:
@alan-c I just tested `kill -USR1 $(<"/var/run/nginx.pid")` outside the script, and it seems to work correctly (with root privilege as expected). Is it working for you outside the script? I didn't want to make it more complicated for you. If it doesn't work inside the script, we could use `cat` as in your initial commit. the performance difference is minimal anyway.